static/app/views/projectInstall/createProject.tsx

```diff
@@ -81,6 +84,52 @@ function CreateProject() {
   const [alertRuleConfig, setAlertRuleConfig] = useState<IssueAlertFragment | undefined>(
     undefined
   );
+  const [alertNotificationAction, setAlertNotificationAction] = useState<
+    IssueAlertActionType[]
+  >([IssueAlertActionType.NOTIFY_EMAIL]);
+  const [alertNotificationProvider, setAlertNotificationProvider] = useState<
+    string | undefined
+  >(undefined);
+  const [alertNotificationIntegration, setAlertNotificationIntegration] = useState<
+    OrganizationIntegration | undefined
+  >(undefined);
+  const [alertNotificationChannel, setAlertNotificationChannel] = useState<
+    string | undefined
+  >(undefined);
```

Comment:
Does all this state belong in the `createProject` component itself? Should this live somewhere more contained?

Reply:
I thought about this too, but I wasn't sure where it would go because the alert rule is created here after the project itself has been created.

I also considered turning provider, integration, and channel into an object, but decided against it since I would just have to create individual setters for readability anyways- lmk if that wasn't the right approach tho